relay-server/src/services/project.rs

```diff
@@ -1174,7 +1196,7 @@ impl Project {
             return CheckedBuckets::ProjectDisabled(len);
         }
 
-        let Some(scoping) = self.scoping() else {
+        let Some(scoping) = self.scoping().or(self.partial_scoping()) else {
```

Comment:
so this is the most crucial line in this PR. If anyone have any ideas why it might be bad to use partial scoping here then please comment it

Reply:
it'd be none if it's sent from the batch_metrics route though, although i think that only gets sent from another relay. I guess it would be possible if you have multiple proxy relays in a chain and they have global metrics enabled?  
 
so we could also create a scoping from just the projectkey here for that scenario